pwa/src/templates/templateParts/cronjobsForm/CreateCronjobsFormTemplate.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import * as styles from "./CronjobsFormTemplate.module.css";
+import { useForm } from "react-hook-form";
+import APIContext from "../../../apiService/apiContext";
+import FormField, { FormFieldInput, FormFieldLabel } from "@gemeente-denhaag/form-field";
+import { Alert, Button, Heading1 } from "@gemeente-denhaag/components-react";
+import { useTranslation } from "react-i18next";
+import APIService from "../../../apiService/apiService";
+import { InputText, Textarea } from "@conduction/components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFloppyDisk } from "@fortawesome/free-solid-svg-icons";
+import { useQueryClient } from "react-query";
+import { useCronjob } from "../../../hooks/cronjob";
+
+interface CreateCronjobFormTemplateProps {
+  cronjobId?: string;
+}
```

Comment:
When _creating_ a Cronjob, we (should) never receive a `cronjobId`, because it doesn't exist yet.

Therefore, these props are redundant.